### PR TITLE
Check if getValue returns true in the constructor

### PR DESCRIPTION
**Pull request by jwendell**

### CODE REVIEW

**Review by rcernich — APPROVED** [review]

File: extensions/common/context.cc [hunk 1]
```diff
@@ -346,8 +346,9 @@ PeerNodeInfo::PeerNodeInfo(const std::string_view peer_metadata_id_key,
   // Attempt to read from filter_state first.
   found_ = getValue({peer_metadata_id_key}, &peer_id_);
   if (found_ && peer_id_ != kMetadataNotFoundValue) {
-    getValue({peer_metadata_key}, &peer_node_);
-    return;
+    if (getValue({peer_metadata_key}, &peer_node_)) {
```
Comment (oschaaf) [inline extensions/common/context.cc:349]: Should we set found_ to false?

Comment (oschaaf) [reply extensions/common/context.cc:349]: Sorry - went out to early. I wonder — should we update found to false when the second getValue fails?

Comment (jwendell) [reply extensions/common/context.cc:349]: It's set to false two lines later

Comment (oschaaf) [reply extensions/common/context.cc:349]: Great, lgtm

**Review by oschaaf — APPROVED** [review]

**Pull request merged**